content/en/releases/patch-releases.md

```diff
@@ -78,9 +78,10 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Cherry Pick Deadline | Target date |
 | --------------------- | -------------------- | ----------- |
-| March 2024            | 2024-03-08           | 2024-03-13  |
-| April 2024            | 2024-04-12           | 2024-04-17  |
+| April 2024            | 2024-04-12           | 2024-04-16  |
 | May 2024              | 2024-05-10           | 2024-05-15  |
+| June 2024              | 2024-06-14           | 2024-06-19  |
+| July 2024              | 2024-07-12           | 2024-07-17  |
```

Comment:
Can we make sure those dates are coordinates with #45770?

Reply:
I aligned June with #45770. 

I dropped July since it wasn't there. April we need to update since we are changing April's date.